android/.gitignore

```diff
@@ -0,0 +1,15 @@
+*.iml
+.gradle
+/local.properties
+/.idea/caches
+/.idea/libraries
```

Comment:
More a general comment: I was already thinking about splitting the repository into Python and ESP, since the PIP package (when cloning from github and then installing) is huge! 
Do you think it would make sense to seperate the client/server architecture into two repos? Having one makes sense if we keep everything close together

Reply:
yes can make sense. but the main reason causing its getting big are the pushed binarys from the esp building process.
would keep this repo and move the esp and python code into git submodules.
then every project have its own repo but you can fork all at once with cloning this one